lib/ht_sip_validator/sip_validator_runner.rb

```diff
@@ -16,22 +18,32 @@ def run_validators_on(sip)
     results = {}
     messages = run_package_checks(sip, results)
 
-    sip.each_file do |filename,filehandle|
+    sip.each_file do |filename, filehandle|
       messages += run_file_checks(filename, filehandle, sip, results)
     end
-    messages.reduce(:+)
+
+    summarize_results(messages)
   end
 
   private
 
+  def summarize_results(messages)
+    flattened_messages = messages.reduce(:+)
+    error_count = flattened_messages.select(&:error?).count
+    warning_count = flattened_messages.select(&:warning?).count
+
+    status = (error_count.zero? ? "Success" : "Failure")
+    puts "#{status}: #{error_count} error(s), #{warning_count} warning(s)"
```

Comment:
Yeah, the `SIPValidatorRunner` should already be returning an array of messages, so I think this formatting/output could move to `ValidateSIPCommand` which would make more sense. I'll make that change.